logging_format/visitor.py

```diff
@@ -238,6 +238,22 @@ def get_except_handler_name(self, node):
             return name.id
         return name
 
+    def get_id_attr(self, value, attempt=0):
```

Comment:
I would now change this code to not recurse:

```
target = value
if not hasattr(target, "id") and hasattr(target, "value"):
    target = target.value
return getattr(value, "id")
```

Reply:
Yes, recursion is useless now, fixed. Please check it